### DISCUSSION TITLE
Modern ToolBar/MenuBar widget

### DISCUSSION THREAD

**Discussion by Akascape** [TomSchimansky/CustomTkinter#1670] [Showcase]
## [CTkMenuBar](https://github.com/Akascape/CTkMenuBar)
This is a modern looking menu bar for customtkinter.

## Features
- Custom dropdown menu
- Add menu in top of title bar
- Classic and modern menubar with all customisability
- Add commands and submenus 

![menubar](https://github.com/Akascape/CTkMenuBar/assets/89206401/02c512b2-557f-4d59-86e0-a6eb9da3696c)

![titlebar](https://github.com/Akascape/CTkMenuBar/assets/89206401/da6cd858-f700-476c-a2f0-93a1c6335a4d)

**Comment by ChromeGenesis** [comment]
### A CTk compatible menu (Mainly for windows using place manager):

[customMenu](https://github.com/ChromeGenesis/customtkinter-menu)

- Every standard option an OG tkinter menu has
- slight caveat might be that the "menu_bar" is placed in the window using the place geometry manager (might raise issues for others using grid etc...)
- Open to better mods for the previous "issue"

![dark_mode](https://github.com/TomSchimansky/CustomTkinter/assets/49604988/5c8fff4f-d607-49f2-9991-98cd0bbd4cd9)
![light_mode](https://github.com/TomSchimansky/CustomTkinter/assets/49604988/f16982cb-1633-4f92-92ca-86d392d921d2)

**Reply by Akascape** [reply]
What is unique about this menu? Is it better than CTkMenubar?

**Reply by ChromeGenesis** [reply]
Wouldn't say better, just simpler.
CTkMenuBar doesn't seem to have a lot of OG tk menu attrs
like radiobtn, checkbtn etc
and when it drops down, instead of staying on top of all widgets, sometimes it "underlaps" (last time i checked)

EDIT:
ahh, maybe i should have made another showcase thread thingy, apologies, github's not exactly my strong forte
